app/lib/features/activities/providers/notifiers/notifications_list_notifier.dart

```diff
@@ -76,17 +71,17 @@ class NotificationsListNotifier extends StateNotifier<NotificationListState>
     _listener = client.notificationsStream();
     if (_listener != null) {
       _poller = _listener!.listen((ev) {
-        debugPrint(
-          ' --- - - ----------------- new notification received',
-        );
+        // debugPrint(
+        //   ' --- - - ----------------- new notification received',
+        // );
 
-        final provider = ref.watch(featuresProvider);
-        if (provider.isActive(LabsFeature.showNotifications)) {
-          if (!ev.read()) {
-            final brief = extractBrief(ev);
-            notify(brief);
-          }
-        }
+        // final provider = ref.watch(featuresProvider);
```

Comment:
Why is this commented out? Or its been moved somewhere else? If it is, you can remove comment.

Reply:
It was some previous code that would show local notifications just coming from our own local supply mechanism and show a local notification regardless of actual push ability on the device ... I suppose we could think about activating something similar again for desktop but we'd have to think about how to phrase that and actually make sure we get the needed data ... I'll remove this for now.